qt/mainwindow.cpp

```diff
@@ -634,7 +634,7 @@ void MainWindow::OnUploadEditsMenuItem()
   {
     auto & editor = osm::Editor::Instance();
     if (editor.HaveMapEditsOrNotesToUpload())
-      editor.UploadChanges(key, secret, {{"created_by", "OMaps " OMIM_OS_NAME}});
+      editor.UploadChanges(key, secret, {{"created_by", "Organic Maps " OMIM_OS_NAME}});
```

Comment:
That change may complicate debugging for osmers.

Reply:
Mobile uses the full string "Organic Maps" eg. https://www.openstreetmap.org/changeset/133341165) so this will actually unify desktop with mobile :)